### DISCUSSION TITLE
To better facilitate support from Chinese users

### DISCUSSION THREAD

**Discussion by sxj1991** [drakkan/sftpgo#1614] [General]
Suggestion:
To better facilitate support from Chinese users, who tend to favor digital payment methods like mobile apps over credit cards, considering integrating with applications such as Alipay would be advantageous. By simply posting your QR code in the sponsorship section, users can swiftly scan and contribute to your project.

Implementation Steps:
Set Up a Receiving Account: Begin by obtaining a personal or business account with Alipay (or WeChat Pay), depending on your project’s needs. For individual projects, a personal account suffices, which can be set up through their official channels.

Generate a Payment QR Code: Within the Alipay (or WeChat Pay) backend, you can easily create a payment QR code that directly links to your account. Ensure the code is configured to accept payments from anyone and customize the payment note to indicate, for example, "Sponsorship for [Project Name]."

Publish the QR Code: Display this QR code prominently on your project’s website, social media profiles, and any promotional materials related to the project. Clearly label it as the dedicated QR code for sponsoring your project and provide a brief guide on how to use it for support.

Transparency & Feedback: To foster trust and encourage further contributions, periodically update sponsors on how funds are being utilized or project advancements. This ensures sponsors feel their contributions are valued and impactful.

Compliance: Throughout the process, adhere to local regulations and the service terms of Alipay and WeChat Pay, particularly concerning transparency in fund management and legal compliance.

Implementing these steps will effectively provide Chinese users with a familiar and convenient means of sponsorship, thereby boosting your project’s funding and growth. Additionally, addressing potential privacy and security concerns by assuring users their information will be protected and not misused is crucial for building trust.

**Comment by drakkan** [comment]
Thanks for your suggestion, this request had already been made in the past. Unfortunately I believe we need to be a resident of China or at least have a bank account in China to be able to activate an Alipay account and receive money. We don't meet this requirement, we are based in Italy. 

Furthermore, GitHub's sponsor program does not support Alipay/WeChat.

However, we were able to get support for Alipay through the Stripe payment gateway which we use for the support plans we offer through our site

https://sftpgo.com/#pricing

Alipay is available for one-time payments, no one has ever used it until now and it required a lot of effort to get approved.

As for transparency & feedback, the funds are used to allow us to work on SFTPGo instead of other paid consultancies and if you look at the commit [history](https://github.com/drakkan/sftpgo/commits/main/) you can see that I work many hours per week on SFTPGo and I also contribute to related projects used in SFTPGo such as crypto/ssh, pkg/sftp, ftpserverlib. 
This allows you all to rely on a production-level file transfer solution comparable to similar proprietary products.

For the new user interfaces that you will find in the upcoming 2.6 version, we also paid for the UI theme and a specialized UX company to improve the user experience as well as investing our time in writing code.

It is very important to understand that you should support open source projects for ongoing maintenance: if SFTPGo got unmaintained and so bug and security issues will not be fixed you will be forced to switch to a different product, probably paying for a proprietary license and afford the migration costs.
Open source maintainers are generally passionate developers who enjoys sharing their work with the community, but if they can't pay their bills the problem could be passed on to you and your company and you will lose money.
